wagtail/snippets/templates/wagtailsnippets/snippets/edit.html

```diff
@@ -20,9 +20,11 @@
                                     <span class="icon icon-spinner"></span><em>{% trans 'Save' %}</em>
                                 </button>
                                 <div class="dropdown-toggle icon icon-arrow-up"></div>
-                                <ul role="menu">
-                                    <li><a href="{% url 'wagtailsnippets:delete' model_opts.app_label model_opts.model_name instance.pk|admin_urlquote %}" class="shortcut">{% trans "Delete" %}</a></li>
-                                </ul>
+                                <nav aria-label="Actions">
```

Comment:
Same as the previous similar example – this should either have no landmark, or have it on the wrapper `ul` above.

Reply:
Moved to surround wrapper `ul`